### ISSUE TITLE
Create chores table and populate it with dummy data

### ISSUE THREAD

**Issue by raghavgarg32** [SOFTENG701G1/Flatmate-Management-System#86]
**User story**
As a Developer, I want to be able to use a chores database table which has dummy data so that I can query it and update it and test functionality.

**Acceptance criteria**
A table of chores is created when the app starts in the database
The table has chores dummy data

**Additional context**
Depends on issue #85

**Comment by lolagarden** [comment]
This issue is already covered by issue #66 'display chores in chore page'. Perhaps a new issue could be made to show dummy data on the chores page but I'm not sure this is needed?

**Comment by HongShi10** [comment]
Is this finished?
